### PR TITLE
Dictionaries update

### PR DESCRIPTION
**Pull request by ratneshraval**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
- Tests will be added to related controller. 

#### Brief feature description
- A different implementation of dictionary. Using Ethnicity and Race dictionary to show. 

#### Does this PR block/effect other stories?
- No 

#### Do any commands needs to be run in order to run the project with this PR?
- No

#### Code coverage percent diff as a result of this PR?
- Ruby 78%, JS same as before. 

#### JS errors diff as a result of this PR?
- Same as before.

### CODE REVIEW

File: app/models/concerns/base_cals_api_protocol_provider.rb [hunk 1]
```diff
@@ -4,12 +4,12 @@ module Concerns::BaseCalsApiProtocolProvider
...
   class_methods do
     def all(auth_header)
-      response = FaradayCals.get('/' + class_name_downcase_pluralized, auth_header)
+      response = FaradayCals.get('/' + api_resource_path, auth_header)
       JSON.parse(response.body)['items'].map { |itm| new(itm) }
     end
 
     def find_by_id(id, auth_header)
-      response = FaradayCals.get("/#{class_name_downcase_pluralized}/#{id}", auth_header)
+      response = FaradayCals.get("/#{api_resource_path}/#{id}", auth_header)
```
Comment (lavanyareddyreddy) [inline app/models/concerns/base_cals_api_protocol_provider.rb]: Glad you replaced it with api_resource_path.

File: app/models/dictionary_base.rb [hunk 2]
```diff
@@ -1,7 +1,11 @@
 class DictionaryBase < CalsBase
-  Concerns::BaseCalsApiProtocolProvider
```
Comment (lavanyareddyreddy) [inline app/models/dictionary_base.rb]: Great, we can also remove concerns namespace at some point by adding it to auto load

File: app/models/helpers/dictionaries/race_type_helper.rb [hunk 3]
```diff
@@ -1,6 +0,0 @@
-class Helpers::Dictionaries::RaceTypeHelper < Helpers::ModelHelperBase
-
-  def model_class
```
Comment (lavanyareddyreddy) [inline app/models/helpers/dictionaries/race_type_helper.rb]: Great that you got rid of these!

File: app/models/dictionary_base.rb (hunk 2 above)
Comment (ratneshraval) [inline app/models/dictionary_base.rb]: Can you create a story for that based on the article you found.

**Pull request merged**